strapi/strapi
# fix: Fix Cursor issue hover over local select list

### What does it do?


### Why is it needed?
To fix issue [21337](https://github.com/strapi/strapi/issues/21337) : Missing the cursor pointer in the list field when switching from one locale to another.

### How to test it?

1. Enable localization for any collection type you want and choose the no. of locales as your need from global settings.
2. Now go to content-manager and open a record from list of any collection-type.
3. You will be able to see the locales option to choose at the top (above save/preview menu options).
4. If you click and hover over the locales options list, you will not get cursor pointer there while hover.  

### Related issue(s)/PR(s)
fixes [21337](https://github.com/strapi/strapi/issues/21337)
